source/base/quadrature.cc

```diff
@@ -79,7 +82,8 @@ template <int dim>
 Quadrature<dim>::Quadrature (const std::vector<Point<dim> > &points)
   :
   quadrature_points(points),
-  weights(points.size(), std::atof("Inf"))
+  weights(points.size(), std::atof("Inf")),
```

Comment:
I missed that this was already there and I would appreciate if you could fix it while we are here.

It looks like that constructor was written in 2001. Perhaps `numeric_limits` was not always available back then.